src/rstar.jl

```diff
@@ -0,0 +1,125 @@
+# Data structure required to store samples for MLJ.
+struct RStarTable{T<:AbstractFloat}
```

Comment:
Why is this needed? Can't you just provide a [`MatrixTable`](https://github.com/JuliaData/Tables.jl/blob/master/src/matrix.jl) to the MLJ model? I.e., users provide the data as matrices to `rstar` and you call `table = Tables.table(data)` internally?

Reply:
Great, I wasn't aware of MatrixTable.